modinfo.lua

```diff
@@ -2566,14 +2565,28 @@ local chinese = {
 			["3"] = {
 				DESCRIPTION = "3秒",
 				HOVER = "信息每3秒更新。"
-			},
-		},
+			}
+		}
 	},
 	--------------------------------------------------------------------------
 	--[[ Debugging ]]
 	--------------------------------------------------------------------------
+	log_reporter = {
```

Comment:
I wonder how long this configuration has been here and has escaped my notice. It should not be present and will be removed in the next update.

Reply:
It appears at Line 1283 in english config but not in chinese, so I thought this was missing.